sematic/ui/tests/support/component-index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
Is this file necessary?

Reply:
Thanks for asking about it.

Yes, very necessary. It is the bootstrap page for the component test. In that, all component tests are run on a real webpage. And this is the webpage.